mahjongiOS/Controller/ViewController.swift

```diff
@@ -92,31 +94,41 @@ class ViewController: UIViewController, GameDelegate {
          tile selected during discard, call/expose and joker trade to be
          decoupled.
         */
-        let button = sender as! UIButton
+        exchangeButton.isEnabled = false
+        discardButton.isEnabled = false
         let tileIndex = button.tag
-        if discardState && selectedIndexes.count > 0 && !button.isSelected {
-            let alert = UIAlertController(title: "Error", message: "Select single tile to discard", preferredStyle: .alert)
-            alert.addAction(UIAlertAction(title: NSLocalizedString("OK", comment: "Default action"), style: .default))
-            self.present(alert, animated: true, completion: nil)
-            return
+        if discardState && selectedIndexes.count == 1 && !button.isSelected {
```

Comment:
Really like the minor change from `> 0` to `== 1`. I was literally just sitting for here a couple minutes trying to remember that this code is for when you're trying to select more than 1 tile. But now it's much more readable